src/stores/federation.js

```diff
@@ -121,7 +121,6 @@ export const useFederationStore = defineStore('federation', {
 			Vue.delete(this.pendingShares[id], 'loading')
 			Vue.set(this.acceptedShares, id, {
 				...this.pendingShares[id],
-				accessToken: conversation.remoteAccessToken,
```

Comment:
cc @Antreesy

Reply:
Used in the store, bu we don't rely on it